packages/cli/src/platform/accelerate/enable.ts

```diff
@@ -1,10 +1,12 @@
 import { arg, Command, isError } from '@prisma/internals'
+import { log } from 'console'
```

Comment:
If you could avoid this and use `console.info`, we don't use this at all in the codebase and it's better to be consistent.